packages/preprocess/src/add-script-tags.cjs

```diff
@@ -5,13 +5,17 @@ const addScriptTags = {
 	markup({ content, filename }) {
 		if (filename.endsWith('.md')) {
 			if (!content.match(/\<script(.*)\>/)) {
-				return { code: '<script context="module"> </script>' + '<script> </script>' + content };
+				const result =
+					content + '\n\n<script context="module"> </script>\n\n<script> </script>\n\n';
+				return { code: result };
 			}
 			if (!content.match(/\<script(.*)context\=\"module\"(.*)\>/)) {
-				return { code: '<script context="module"> </script>' + content };
+				const result = content + '\n\n<script context="module"> </script>\n\n';
+				return { code: result };
 			}
 			if (!content.match(/\<script\>/)) {
-				return { code: '<script> </script>' + content };
+				const result = content + '\n\n<script> </script>\n\n';
+				return { code: result };
```

Comment:
What was the reason for these changes?

Reply:
Preprocess was working a little differently, there were some minor syntax changes needed to get it functioning properly again